src/pi-src/uid.cpp

```diff
@@ -0,0 +1,55 @@
+/******************************************************************************
+Decodes unique identifiers from transmissions
+
+The details of UID encoding and decoding can be found in
+.../doc/TransmissionProtocol/TransmissionProtocol.pdf,
+also known as the iCRAB Protocol Documentation.
```

Comment:
This should be an absolute path not a relative one.

Reply:
Hrmm. I like to keep the source code somewhat separate so that it or the documentation can move independently of each other, but I realize that I'm in a weird middle ground by doing it the way that I am in the code. I'll change all of these to just be the document title so it's future proofed. Thanks for bringing this up